tracer/build/_build/Build.Profiler.Steps.cs

```diff
@@ -38,9 +39,88 @@ partial class Build
         .DependsOn(CompileProfilerNativeTestsWindows)
         .DependsOn(CompileProfilerNativeTestsLinux);
 
+    Target SetupVcpkg => _ => _
+        .Unlisted()
+        .OnlyWhenStatic(() => IsWin)
+        .Executes(async () =>
+        {
+            var vcpkg = await GetVcpkg();
+
+            var vcpkgObjFolder = NativeBuildDirectory / "vcpkg";
+            var buildtreesRoot = vcpkgObjFolder / "buildtrees";
+            var packagesRoot = vcpkgObjFolder / "packages";
+            var downloadRoot = vcpkgObjFolder / "download";
+            vcpkg($"integrate install --x-buildtrees-root={buildtreesRoot} --x-packages-root={packagesRoot} --downloads-root={downloadRoot}");
+
+            async Task<Tool> GetVcpkg()
```

Comment:
This is obvs all windows-only - do we not need this to work on linux too? Or is it baked-in to cmake?

Reply:
For now, I just want windows only. For CMake, there is a bit setup + command-line update.
Not a big deal, but want to scope the change to windows first